### DISCUSSION TITLE
proptest

### DISCUSSION THREAD

**Discussion by jphughes9** [statdivlab/rigr#94] [General]
@cwolock Can proptest handle summary statistics (like binom.test and prop.test) only variables. e.g. 
prop.test(67,114,p=.5)

**Comment by cwolock** [comment]
Not `proptest`, but `proptesti`, yes.

**Comment by jphughes9** [comment]
Ah, great … sorry I missed that.

Jim
--------------
Jim Hughes
Professor of Biostatistics
University of Washington           Email: ***@***.******@***.***>
Mailstop 351617                           Phone: 206-616-2721
Seattle, WA  98195                        Fax:      206-616-2724
                                                          Cell:      206-498-2456
